### PR TITLE
add styling

### PR DESCRIPTION
**Pull request by jellywiz**
### Project requirements

- At this point you should have a fully working app that uses only basic styles to make the layout work, according to this wireframe

- Now you should improve the *look and feel* of the application, adding the styles of your choice.
- Please keep the general layout of the wireframe, this is the only mandatory requirement.
- You can use plain CSS or any CSS framework, it's up to you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 # Live Demo
 
-[Demo](Link)
+[Demo](https://jellywiz.github.io/Leaderboard/)
 
 ## Getting Started
```
Comment (OLIPLICHE) [inline README.md]: - **Please make your README file look professional:** It should contain all that's needed for running the app with the API(s) it consumes. 
**Hint**: How to make the project run on your local machine(You will need to set up or install webpack in your machine and also talk about the installation of Package.Json)
-
-
Capture

File: src/style.css [hunk 2]
```diff
@@ -1,18 +1,26 @@
+body {
+  background-color: #8ebfe7;
+}
+
 header {
-  background-color: #000;
```
Comment (OLIPLICHE) [inline src/style.css:2]: - Nice Job 🥇 🥇🥇

Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @jellywiz, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- App is properly styled 👏👏
- Linter is green (the last commit has a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Gitflow is correctly used (Pull Request is made from the feature branch into the dev branch) 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 2 above)
Comment (jellywiz) [reply src/style.css:2]: do i need to change anything on that? @OLIPLICHE

File: README.md (hunk 1 above)
Comment (jellywiz) [reply README.md]: so i should delete this installation part? @OLIPLICHE

File: src/style.css (hunk 2 above)
Comment (OLIPLICHE) [reply src/style.css:2]: -Meet me on slack

**Review by OLIPLICHE — APPROVED** [review]
Hi @jellywiz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Implemented the change👏 👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**